### DISCUSSION TITLE
A general remix's headers handler

### DISCUSSION THREAD

**Discussion by nichtsam** [epicweb-dev/epic-stack#809] [General]
I noticed that `epic-stack` doesn't have utils like `reuseUsefulLoaderHeaders` that Kent has on `kentcdodds.com`. 
I thought it'd be a nice thing to have, it's kind of a pain to manage all that headers stuff, so I integrated it into my project with some modifications. 
It has been working nicely on my project for a while already, and I thought that `epic-stack` might be able to benefit from this.

Here is the gist for the code I've been using:
https://gist.github.com/nichtsam/bff3a0e58f7edfad3985e9322582f67f

**Comment by kentcdodds** [comment]
Thanks @nichtsam! I think that could be useful as well. I am open to a PR!

**Reply by nichtsam** [reply]
PR opened! #810
